### PR TITLE
feat: private network smart contract example

### PR DESCRIPTION
**Pull request by JustinCast**

### CODE REVIEW

File: .vscode/c_cpp_properties.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
```
Comment (xavier506) [inline .vscode/c_cpp_properties.json:1]: should we add this file to the `.gitignore` ?

perhaps if there is an EOS studio project file we can include it in the repo.

File: PrivateNetworkExample/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# easycontract
```
Comment (xavier506) [inline PrivateNetworkExample/README.md]: nice name

File: PrivateNetworkExample/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+### Create key pair
+Execute `cleos create key --to-console` and save the output. (you can also use `--file filename` to save the output in a specific "filename" provided)
+
+### Create Local Wallet
+Execute `cleos wallet create -n easycontract --to-console` and save the output password (you can also use `-f easycontractpass.txt`)
+
+### Import private key
+`cleos wallet import -n easycontract --private-key YOUR_PRIVATE_KEY_HERE`
+
+## Compile easycontract
```
Comment (xavier506) [inline PrivateNetworkExample/README.md:12]: Should we add another step called 

## Deploy Contract

File: PrivateNetworkExample/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+`eosio-cpp -abigen -I include -R resource -contract easycontract -o easycontract.wasm src/easycontract.cpp`
```
Comment (xavier506) [inline PrivateNetworkExample/README.md]: Add another step called 

`## Deploy Contract`

File: PrivateNetworkExample/README.md (hunk 2 above)
Comment (xavier506) [reply PrivateNetworkExample/README.md]: Lets add these files in a directory called `easycontract`

File: .vscode/c_cpp_properties.json (hunk 1 above)
Comment (JustinCast) [reply .vscode/c_cpp_properties.json:1]: Adding to .gitignore now...
I didn't use an EOS Studio project. just the regular way

File: PrivateNetworkExample/README.md (hunk 4 above)
Comment (JustinCast) [reply PrivateNetworkExample/README.md]: Ok

File: PrivateNetworkExample/README.md (hunk 2 above)
Comment (JustinCast) [reply PrivateNetworkExample/README.md]: **easycontract** Instead of **PrivateNetworkExample** ?

Comment (xavier506) [reply PrivateNetworkExample/README.md]: Yes

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**